### PR TITLE
Delete Localization groups

### LINKED ISSUE #8692: Delete localization access groups
Localization groups and their permissions are no longer used, let's get rid of the numerous access groups.

### PR DESCRIPTION
**Pull request by wagnerand**
Fix #8692.

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
Did we find another alternative to expose all the helpful localizers that help translating AMO? Afaik these groups were used on a page (that may have been removed in the latest pages-purge) that listed all localizers.

Or do we simply ignore that now?

**Comment by wagnerand** [comment]
Are you talking about the credits page? That one is gone, yes.

**Comment by EnTeQuAk** [comment]
> Are you talking about the credits page? That one is gone, yes.

Yes, I am.

> Also, those groups are maintained manually and they might or might not reflect the current localizers.

True, I do remember a few cases while I'm working on AMO where localizers asked to be on that list because they contributed. We may want to think about an alternative list to honor them, maybe something for @caitmuenster to think about :)

File: src/olympia/migrations/1029-delete-localization-groups.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DELETE FROM groups_users WHERE group_id IN (SELECT id FROM groups WHERE name LIKE "% Localizers");
+DELETE FROM groups WHERE name LIKE "% Localizers";
```
Comment (EnTeQuAk) [inline src/olympia/migrations/1029-delete-localization-groups.sql]: I was a bit confused about the whitespace so for reference, this is how the groups were referenced:

``.filter(groupuser__group__name='%s Localizers' % lang)`` so this is fine.

Comment (EnTeQuAk) [reply src/olympia/migrations/1029-delete-localization-groups.sql]: Verified both queries on sql.telemetry.mozilla.org, looks perfect.

**Comment by caitmuenster** [comment]
Whoa, I had no idea there was a localizers group. 

I'm definitely interested in figuring out how to capture localization contributions to AMO so we can list them on our [Recognition Wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). @wagnerand, is that something we can add to the Contribution Bot?

**Comment by wagnerand** [comment]
We could hack something for the bot, but it seems pontoon already has a nice contributor overview, here is the one for addons-server, for example: https://pontoon.mozilla.org/projects/amo/contributors/

Would that suffice?

**Pull request merged**